core/server/inventory.go

```diff
@@ -71,6 +71,10 @@ func (cs *coreServer) getKustomizationInventory(ctx context.Context, clusterName
 		return nil, fmt.Errorf("failed to get kustomization: %w", err)
 	}
 
+	if kust.Status.Inventory == nil {
+		return []*pb.InventoryEntry{}, nil
+	}
```

Comment:
I think this can return `nil` here?

The PB generates `omitempty` so either way the result would be the same?

https://github.com/weaveworks/weave-gitops/blob/main/pkg/api/core/core.pb.go#L114

If you're not sure if this results in the same thing, https://go.dev/play/p/Av1DA7zf9QF